custom_dc/database_update.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: Update your database schema
+nav_order: 9
+parent: Build your own Data Commons
+---
+
+{:.no_toc}
+# Update your database schema
+
+While starting Data Commons services, you may see an error that starts with `SQL schema check failed`. This means your database schema must be updated for compatibility with the latest Data Commons services.
+
+You can update your database by running a data management job with the environment variable `SCHEMA_UPDATE_ONLY` set to `true`. This will alter your database without modifying already-imported data.
```

Comment:
May need to be updated based on what we decide in https://github.com/datacommonsorg/website/pull/4686/files#r1815646141

Reply:
Updated.